client/locations-list-map.js

```diff
@@ -208,11 +208,11 @@ function createFilters(data) {
     states: {},
     entryAge: {},
   };
-  filters.entryAge['1-7'] = { name: '1-7 days ago', isSet: false, value: '1-7' };
-  filters.entryAge['8-14'] = { name: '8-14 days ago', isSet: false, value: '8-14' };
-  filters.entryAge['15-21'] = { name: '15-21 days ago', isSet: false, value: '15-21' };
-  filters.entryAge['21+'] = { name: '21+ days ago', isSet: false, value: '21' };
-  filters.entryAge.placeholder = 'Last Updated';
+  filters.entryAge['1-7'] = { name: $.i18n('ftm-entry-age-1-7'), isSet: false, value: '1-7' };
```

Comment:
I'll merge in smartling translations to the pr, agree a for loop isn't really necessary but maybe just declare as one object?

```
entryAge: {
  '1-7': { name: $.i18n('ftm-entry-age-1-7'), isSet: false, value: '1-7' },
  ...
}
```